### PR TITLE
[ES, C*] Check for the presence of a tag

### PR DESCRIPTION
**Pull request by annanay25**
## Which problem is this PR solving?
- Adds functionality to check for the presence of a tag, to the ES and C* readers.

## Short description of the changes
- New queries for C* and ES backends.
- Added parameter to the GRPC TraceQueryParameters and updated proto-generated files.
- Added tests.

In the UI, this feature could be implemented as a search-box, input being a comma separated list of tags that need to be present in the result traces.

### CODE REVIEW

File: plugin/storage/cassandra/spanstore/reader.go [hunk 1]
```diff
@@ -301,9 +308,30 @@ func (s *SpanReader) findTraceIDs(ctx context.Context, traceQuery *spanstore.Tra
 	if len(traceQuery.Tags) > 0 {
 		return s.queryByTagsAndLogs(ctx, traceQuery)
 	}
+	if len(traceQuery.CheckTagsPresent) > 0 {
+		s.queryCheckTagPresence(ctx, traceQuery)
```
Comment (yurishkuro) [inline plugin/storage/cassandra/spanstore/reader.go]: Return?

Comment (annanay25) [reply plugin/storage/cassandra/spanstore/reader.go]: Oops. Will fix.

**Comment by annanay25** [comment]
I'd like to wait for https://github.com/jaegertracing/jaeger/pull/2049 to be merged before moving forward with this.

**Comment by jpkrohling** [comment]
@annanay25 looks like #2049 is already there ;-)

**Pull request closed without merging**